release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -500,6 +500,15 @@ $ oc patch featuregates cluster -p '{"spec": {"featureSet": "TechPreviewNoUpgrad
 [id="ocp-release-notes-scalability-and-performance_{context}"]
 === Scalability and performance
 
+[id="ocp-release-notes-node-tuning-operator-deferred-updates_{context}"]
+==== Node Tuning Operator: Deferred Tuning Updates
+
+In this release, the Node Tuning Operator introduces support for deferring tuning updates. This feature allows administrators to schedule updates to be applied during a designated maintenance window.
+
+For latency-sensitive applications that cannot tolerate the removal and reapplication of tuning updates, a new annotation, `tuned.openshift.io/deferred``, has been introduced. When this annotation is set to `always`, it ensures that any changes are deferred until after the next reboot.
```

Comment:
Should we give an example in the release note or just use the more info link ?  and if so , do we need to specify the default already ?  (just thinking , not suggesting at the moment)